docs/book/src/cronjob-tutorial/testdata/project/internal/webhook/v1/webhook_suite_test.go

```diff
@@ -30,11 +30,7 @@ import (
 	. "github.com/onsi/gomega"
 
 	admissionv1 "k8s.io/api/admission/v1"
-
-	batchv1 "tutorial.kubebuilder.io/project/api/v1"
-
-	// +kubebuilder:scaffold:imports
-	apimachineryruntime "k8s.io/apimachinery/pkg/runtime"
```

Comment:
was this one not used?
If so good catcher.

Reply:
it was

I removed because webhooks suite test used an "empty" scheme (with only user CRDs) and controllers suite test used a "fully loaded" scheme (with user CRDs and default objects loaded, like namespaces, pods, etc)

Without this change, users can not create a namespace in webhook envtest tests, for example

Reply:
To ensure clarity and make it easier to follow the changes, I think we should split this into separate PRs for each goal. This approach will help in maintaining a clear discussion for each change, simplify the release notes, and ensure each fix or change gets the attention it deserves.

Could you please open a PR for each fix/change proposed?

Reply:
will do

Reply:
Just to clarify: For example, in this case, we need to explain what has changed and why in the PR, and release notes so due that we cannot have many changes with diff purposes at the same PR